packages/airnode-validator/src/config/config.ts

```diff
@@ -62,7 +64,8 @@ export const chainTypeSchema = z.literal('evm');
 
 export const airnodeRrpContractSchema = z
   .object({
-    AirnodeRrp: evmAddressSchema,
+    AirnodeRrp: evmAddressSchema.optional(),
```

Comment:
Here is a clarification to make it clearer: Although AirnodeRrp shouldn't be optional, I have set both contracts to be optional in this case. The reason behind this decision is to accommodate situations where the user wants to use their own address for one of the contracts, while using default deployments for the other contract. In such cases, the user can define chains.contracts for the desired contract but leave it undefined for the contract they wish to use default deployments for. 

Actually this schema is input schema (see [here](https://github.com/colinhacks/zod#type-inference)) and supposed to parse input config.json. Enforcing this schema with transforms below, inferred output type for this schema will be {AirnodeRrp: evmAddressSchema, AirnodeRrpDryRun: evmAddressSchema.optional()}